### PR TITLE
webapp/partials/*.jsp: change to 2-space indentation #7967

### PR DESCRIPTION
**Pull request by sujeet14108**
Fixes partially #7352

### CODE REVIEW

**Comment by sujeet14108** [comment]
@chowyb Please review :)

File: src/main/webapp/partials/instructorHelpGettingStarted.jsp [hunk 1]
```diff
@@ -1,435 +1,438 @@
...
+              <br>
+
+            </li>
+            <br>
+            <li>You can set the visibility options for each question:
+              <br>Allows you to set who can see the answers, giver name and recipient name for each question.
+
+              <br>
+              <br>See
+              <a href="#sessionTypes">here</a> for more info about session types.</li>
```
Comment (chowyb) [inline src/main/webapp/partials/instructorHelpGettingStarted.jsp]: Move `</li>` to the next line.

File: src/main/webapp/partials/instructorHelpProfile.jsp [hunk 2]
```diff
@@ -1,201 +1,201 @@
...
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-xs-12">
+            <div class="panel panel-default">
+              <div class="panel-body">
+                <span data-toggle="modal" data-target="#studentProfileMoreInfo"
+                  class="text-muted pull-right glyphicon glyphicon-resize-full cursor-pointer"></span>
```
Comment (chowyb) [inline src/main/webapp/partials/instructorHelpProfile.jsp]: Add 2 spaces to this line.

**Comment by sujeet14108** [comment]
Ready for review :)

**Pull request merged**